### ISSUE TITLE
Bundler::HTTPError Net::HTTPUnprocessableEntity: Too many gems (use --full-index instead)

### ISSUE THREAD

**Issue by alexfalkowski** [bundler/bundler#3367]
Hi Team,

When I run:

```ruby
bundle outdated
```

The process just hangs reporting the following error:

```ruby
Fetching gem metadata from https://rails-assets.org/...
Fetching gem metadata from https://rubygems.org/...Retrying dependency api due to error (2/3): Bundler::HTTPError Net::HTTPUnprocessableEntity: Too many gems (use --full-index instead)
Retrying dependency api due to error (3/3): Bundler::HTTPError Net::HTTPUnprocessableEntity: Too many gems (use --full-index instead)
```

Here is the output from running bundle env

```ruby
Bundler 1.7.12
Ruby 2.2.0 (2014-12-25 patchlevel 0) [x86_64-darwin14]
Rubygems 2.4.5
rvm 1.26.7 (master)
GEM_HOME /Users/alex/.rvm/gems/ruby-2.2.0@iteach
GEM_PATH /Users/alex/.rvm/gems/ruby-2.2.0@iteach:/Users/alex/.rvm/gems/ruby-2.2.0@global


Gemfile
source 'https://rubygems.org'
source 'https://rails-assets.org'

gem 'rails', '4.1.9'
gem 'rake', '10.4.2'
gem 'jquery-rails', '3.1.2'
gem 'jquery-ui-rails', '4.2.1'
gem 'ancestry', '2.1.0'
gem 'audited-activerecord', '4.0.0'
gem 'acts_as_list', '0.6.0'
gem 'fastercsv', '1.5.5'
gem 'devise', '3.4.1'
gem 'devise_ldap_authenticatable', '0.8.1'
gem 'devise-encryptable', '0.2.0'
gem 'cancancan', '1.10.1'
gem 'princely', '1.4.1'
gem 'progressbar', '0.21.0'
gem 'sidekiq', '3.3.0'
gem 'sidekiq-cron', '0.2.0'
gem 'rufus-scheduler', '3.0.9'
gem 'wisper-sidekiq', '0.0.1'
gem 'wisper-activerecord', '0.2.0'
gem 'sinatra', '1.4.5'
gem 'nokogiri', '1.6.6.1'
gem 'namae', '0.9.3'
gem 'activerecord-oracle_enhanced-adapter', '1.5.5'
gem 'ruby-oci8', '2.1.7'
gem 'ckeditor', '4.1.1'
gem 'rails_autolink', '1.1.6'
gem 'sanitize', '3.1.0'
gem 'faraday', '0.9.1'
gem 'httpclient', '2.6.0.1'
gem 'will_paginate', '3.0.7'
gem 'bootstrap-will_paginate', '0.0.10'
gem 'font-awesome-sass', '3.2.1.4'
gem 'js-routes', '1.0.0'
gem 'gon', '5.2.3'
gem 'dalli', '2.7.2'
gem 'slim-rails', '2.1.5'
gem 'exception_notification', '4.0.1'
gem 'newrelic_rpm', '3.9.9.275'
gem 'newrelic-redis', '1.5.0'
gem 'active_model_serializers', '0.9.3'
gem 'sass', '3.2.19'
gem 'sass-rails', '4.0.5'
gem 'compass-rails', '2.0.0'
gem 'bootstrap-sass', '3.3.3'
gem 'uglifier', '2.7.0'
gem 'yui-compressor', '0.12.0'
gem 'therubyracer', '0.12.1'
gem 'coffee-rails', '4.1.0'
gem 'ngannotate-rails', '0.14.1'
gem 'rails-assets-underscore', '1.7.0'
gem 'rails-assets-angular', '1.2.26'
gem 'rails-assets-angular-sanitize', '1.2.26'
gem 'rails-assets-angular-resource', '1.2.26'
gem 'rails-assets-angular-bootstrap', '0.12.0'
gem 'rails-assets-jquery-treetable', '3.2.0'
gem 'rails-assets-jquery-flot', '0.8.3'
gem 'rails-assets-jquery-hoverIntent-jmalonzo', '0.7.1'
gem 'rails-assets-ui-autocomplete', '0.5.7'
gem 'rails-assets-smartmenus', '0.9.7'
gem 'rails-assets-asquera--prism', '0.0.8'
gem 'roadie-rails', '1.0.4'
gem 'flip', '1.0.1'
gem 'wicked', '1.1.0'
gem 'bcrypt', '3.1.9'

group :development do
  gem 'spring', '1.2.0'
  gem 'capistrano', '2.15.5'
  gem 'capistrano-rails', '1.0.0'
  gem 'net-ssh', '2.7.0'
  gem 'net-scp', '1.1.2'
  gem 'better_errors', '2.1.1'
  gem 'brakeman', '3.0.0', require: false
  gem 'dawnscanner', '1.2.99', require: false
  gem 'parser', '2.2.0.2', require: false
  gem 'rubocop', '0.28.0', require: false
end

group :test, :development do
  gem 'rspec-rails', '3.1.0'
  gem 'rspec-activemodel-mocks', '1.0.1'
  gem 'rspec-its', '1.1.0'
  gem 'jasmine', '2.1.0'
  gem 'cucumber-rails', '1.4.2', require: false
  gem 'pdf-inspector', '1.2.0'
  gem 'database_cleaner', '1.4.0'
  gem 'selenium-webdriver', '2.44.0'
  gem 'capybara', '2.4.4'
  gem 'capybara-webkit', '1.4.0'
  gem 'simplecov', '0.9.1', require: false
  gem 'pry', '0.10.1'
  gem 'pry-byebug', '2.0.0'
  gem 'awesome_print', '1.6.1'
  gem 'timecop', '0.7.1'
  gem 'headless', '1.0.2', require: false
  gem 'factory_girl_rails', '4.5.0', require: false
  gem 'poltergeist', '1.5.1'
  gem 'webmock', '1.20.4', require: false
  gem 'phantomjs', '1.9.8.0'
  gem 'rails-assets-angular-mocks', '1.2.26'
  gem 'ladle', '1.0.0'
  gem 'email_spec', '1.6.0'
  gem 'letter_opener', '1.3.0'
end


Gemfile.lock
GEM
  remote: https://rubygems.org/
  remote: https://rails-assets.org/
  specs:
    Ascii85 (1.0.2)
    actionmailer (4.1.9)
      actionpack (= 4.1.9)
      actionview (= 4.1.9)
      mail (~> 2.5, >= 2.5.4)
    actionpack (4.1.9)
      actionview (= 4.1.9)
      activesupport (= 4.1.9)
      rack (~> 1.5.2)
      rack-test (~> 0.6.2)
    actionview (4.1.9)
      activesupport (= 4.1.9)
      builder (~> 3.1)
      erubis (~> 2.7.0)
    active_model_serializers (0.9.3)
      activemodel (>= 3.2)
    activemodel (4.1.9)
      activesupport (= 4.1.9)
      builder (~> 3.1)
    activerecord (4.1.9)
      activemodel (= 4.1.9)
      activesupport (= 4.1.9)
      arel (~> 5.0.0)
    activerecord-oracle_enhanced-adapter (1.5.5)
    activesupport (4.1.9)
      i18n (~> 0.6, >= 0.6.9)
      json (~> 1.7, >= 1.7.7)
      minitest (~> 5.1)
      thread_safe (~> 0.1)
      tzinfo (~> 1.1)
    acts_as_list (0.6.0)
      activerecord (>= 3.0)
    addressable (2.3.6)
    afm (0.2.2)
    ancestry (2.1.0)
      activerecord (>= 3.0.0)
    arel (5.0.1.20140414130214)
    ast (2.0.0)
    astrolabe (1.3.0)
      parser (>= 2.2.0.pre.3, < 3.0)
    audited (4.0.0)
      rails-observers (~> 0.1.2)
    audited-activerecord (4.0.0)
      activerecord (~> 4.0)
      audited (= 4.0.0)
    autoprefixer-rails (5.0.0.3)
      execjs
      json
    awesome_print (1.6.1)
    bcrypt (3.1.9)
    better_errors (2.1.1)
      coderay (>= 1.0.0)
      erubis (>= 2.6.6)
      rack (>= 0.9.0)
    bootstrap-sass (3.3.3)
      autoprefixer-rails (>= 5.0.0.1)
      sass (>= 3.2.19)
    bootstrap-will_paginate (0.0.10)
      will_paginate
    brakeman (3.0.0)
      erubis (~> 2.6)
      fastercsv (~> 1.5)
      haml (>= 3.0, < 5.0)
      highline (~> 1.6.20)
      multi_json (~> 1.2)
      ruby2ruby (~> 2.1.1)
      ruby_parser (~> 3.5.0)
      sass (~> 3.0)
      slim (>= 1.3.6, < 3.0)
      terminal-table (~> 1.4)
    builder (3.2.2)
    byebug (3.5.1)
      columnize (~> 0.8)
      debugger-linecache (~> 1.2)
      slop (~> 3.6)
    cancancan (1.10.1)
    capistrano (2.15.5)
      highline
      net-scp (>= 1.0.0)
      net-sftp (>= 2.0.0)
      net-ssh (>= 2.0.14)
      net-ssh-gateway (>= 1.1.0)
    capistrano-rails (1.0.0)
      capistrano
    capybara (2.4.4)
      mime-types (>= 1.16)
      nokogiri (>= 1.3.3)
      rack (>= 1.0.0)
      rack-test (>= 0.5.4)
      xpath (~> 2.0)
    capybara-webkit (1.4.0)
      capybara (>= 2.3.0, < 2.5.0)
      json
    celluloid (0.16.0)
      timers (~> 4.0.0)
    childprocess (0.5.5)
      ffi (~> 1.0, >= 1.0.11)
    chunky_png (1.3.3)
    ckeditor (4.1.1)
      cocaine
      orm_adapter (~> 0.5.0)
    climate_control (0.0.3)
      activesupport (>= 3.0)
    cliver (0.3.2)
    cocaine (0.5.5)
      climate_control (>= 0.0.3, < 1.0)
    coderay (1.1.0)
    codesake-commons (1.0.0)
      mechanize
      nokogiri
      rainbow (~> 2.0.0)
    coffee-rails (4.1.0)
      coffee-script (>= 2.2.0)
      railties (>= 4.0.0, < 5.0)
    coffee-script (2.3.0)
      coffee-script-source
      execjs
    coffee-script-source (1.8.0)
    columnize (0.9.0)
    compass (0.12.7)
      chunky_png (~> 1.2)
      fssm (>= 0.2.7)
      sass (~> 3.2.19)
    compass-rails (2.0.0)
      compass (>= 0.12.2)
    connection_pool (2.1.1)
    coveralls (0.7.3)
      multi_json (~> 1.10)
      rest-client (~> 1.7)
      simplecov (~> 0.9.1)
      term-ansicolor (~> 1.3)
      thor (~> 0.19.1)
    crack (0.4.2)
      safe_yaml (~> 1.0.0)
    crass (1.0.1)
    css_parser (1.3.5)
      addressable
    cucumber (1.3.18)
      builder (>= 2.1.2)
      diff-lcs (>= 1.1.3)
      gherkin (~> 2.12)
      multi_json (>= 1.7.5, < 2.0)
      multi_test (>= 0.1.1)
    cucumber-rails (1.4.2)
      capybara (>= 1.1.2, < 3)
      cucumber (>= 1.3.8, < 2)
      mime-types (>= 1.16, < 3)
      nokogiri (~> 1.5)
      rails (>= 3, < 5)
    cvss (0.99.0)
    dalli (2.7.2)
    database_cleaner (1.4.0)
    dawnscanner (1.2.99)
      codesake-commons (~> 1.0.0)
      coveralls
      cvss
      grit
      haml
      justify
      parser
      ptools
      ruby_parser
      sys-uname
      terminal-table
    debugger-linecache (1.2.0)
    devise (3.4.1)
      bcrypt (~> 3.0)
      orm_adapter (~> 0.1)
      railties (>= 3.2.6, < 5)
      responders
      thread_safe (~> 0.1)
      warden (~> 1.2.3)
    devise-encryptable (0.2.0)
      devise (>= 2.1.0)
    devise_ldap_authenticatable (0.8.1)
      devise (>= 3.0)
      net-ldap (>= 0.3.1, < 0.6.0)
    diff-lcs (1.2.5)
    docile (1.1.5)
    domain_name (0.5.23)
      unf (>= 0.0.5, < 1.0.0)
    email_spec (1.6.0)
      launchy (~> 2.1)
      mail (~> 2.2)
    erubis (2.7.0)
    exception_notification (4.0.1)
      actionmailer (>= 3.0.4)
      activesupport (>= 3.0.4)
    execjs (2.2.2)
    factory_girl (4.5.0)
      activesupport (>= 3.0.0)
    factory_girl_rails (4.5.0)
      factory_girl (~> 4.5.0)
      railties (>= 3.0.0)
    faraday (0.9.1)
      multipart-post (>= 1.2, < 3)
    fastercsv (1.5.5)
    ffi (1.9.6)
    flip (1.0.1)
      activesupport (>= 3.0, < 5)
      i18n
    font-awesome-sass (3.2.1.4)
      sass-rails (>= 3.1.1)
    fssm (0.2.10)
    gherkin (2.12.2)
      multi_json (~> 1.3)
    gon (5.2.3)
      actionpack (>= 2.3.0)
      json
      multi_json
      request_store (>= 1.0.5)
    grit (2.5.0)
      diff-lcs (~> 1.1)
      mime-types (~> 1.15)
      posix-spawn (~> 0.3.6)
    haml (4.0.6)
      tilt
    hashery (2.1.1)
    headless (1.0.2)
    highline (1.6.21)
    hike (1.2.3)
    hitimes (1.2.2)
    http-cookie (1.0.2)
      domain_name (~> 0.5)
    httpclient (2.6.0.1)
    i18n (0.7.0)
    jasmine (2.1.0)
      jasmine-core (~> 2.1.0)
      phantomjs
      rack (>= 1.2.1)
      rake
    jasmine-core (2.1.3)
    jquery-rails (3.1.2)
      railties (>= 3.0, < 5.0)
      thor (>= 0.14, < 2.0)
    jquery-ui-rails (4.2.1)
      railties (>= 3.2.16)
    js-routes (1.0.0)
      railties (>= 3.2)
      sprockets-rails
    json (1.8.2)
    justify (1.0.2)
    ladle (1.0.0)
      open4 (~> 1.0)
    launchy (2.4.3)
      addressable (~> 2.3)
    letter_opener (1.3.0)
      launchy (~> 2.2)
    libv8 (3.16.14.7)
    mail (2.6.3)
      mime-types (>= 1.16, < 3)
    mechanize (2.7.2)
      domain_name (~> 0.5, >= 0.5.1)
      http-cookie (~> 1.0.0)
      mime-types (~> 1.17, >= 1.17.2)
      net-http-digest_auth (~> 1.1, >= 1.1.1)
      net-http-persistent (~> 2.5, >= 2.5.2)
      nokogiri (~> 1.4)
      ntlm-http (~> 0.1, >= 0.1.1)
      webrobots (>= 0.0.9, < 0.2)
    method_source (0.8.2)
    mime-types (1.25.1)
    mini_portile (0.6.2)
    minitest (5.5.1)
    multi_json (1.10.1)
    multi_test (0.1.1)
    multipart-post (2.0.0)
    namae (0.9.3)
    net-http-digest_auth (1.4)
    net-http-persistent (2.9.4)
    net-ldap (0.5.1)
    net-scp (1.1.2)
      net-ssh (>= 2.6.5)
    net-sftp (2.1.2)
      net-ssh (>= 2.6.5)
    net-ssh (2.7.0)
    net-ssh-gateway (1.2.0)
      net-ssh (>= 2.6.5)
    netrc (0.10.2)
    newrelic-redis (1.5.0)
      newrelic_rpm (~> 3.0)
      redis (< 4.0)
    newrelic_rpm (3.9.9.275)
    ngannotate-rails (0.14.1)
      execjs
      rails (>= 3.1)
    nokogiri (1.6.6.1)
      mini_portile (~> 0.6.0)
    nokogumbo (1.2.0)
      nokogiri
    ntlm-http (0.1.1)
    open4 (1.3.4)
    orm_adapter (0.5.0)
    parser (2.2.0.2)
      ast (>= 1.1, < 3.0)
    pdf-inspector (1.2.0)
      pdf-reader (~> 1.0)
    pdf-reader (1.3.3)
      Ascii85 (~> 1.0.0)
      afm (~> 0.2.0)
      hashery (~> 2.0)
      ruby-rc4
      ttfunk
    phantomjs (1.9.8.0)
    poltergeist (1.5.1)
      capybara (~> 2.1)
      cliver (~> 0.3.1)
      multi_json (~> 1.0)
      websocket-driver (>= 0.2.0)
    posix-spawn (0.3.9)
    powerpack (0.0.9)
    princely (1.4.1)
    progressbar (0.21.0)
    pry (0.10.1)
      coderay (~> 1.1.0)
      method_source (~> 0.8.1)
      slop (~> 3.4)
    pry-byebug (2.0.0)
      byebug (~> 3.4)
      pry (~> 0.10)
    ptools (1.3.2)
    rack (1.5.2)
    rack-protection (1.5.3)
      rack
    rack-test (0.6.3)
      rack (>= 1.0)
    rails (4.1.9)
      actionmailer (= 4.1.9)
      actionpack (= 4.1.9)
      actionview (= 4.1.9)
      activemodel (= 4.1.9)
      activerecord (= 4.1.9)
      activesupport (= 4.1.9)
      bundler (>= 1.3.0, < 2.0)
      railties (= 4.1.9)
      sprockets-rails (~> 2.0)
    rails-assets-angular (1.2.26)
    rails-assets-angular-bootstrap (0.12.0)
      rails-assets-angular (>= 1, < 1.3.0)
    rails-assets-angular-mocks (1.2.26)
      rails-assets-angular (= 1.2.26)
    rails-assets-angular-resource (1.2.26)
      rails-assets-angular (= 1.2.26)
    rails-assets-angular-sanitize (1.2.26)
      rails-assets-angular (= 1.2.26)
    rails-assets-asquera--prism (0.0.8)
    rails-assets-jquery (1.9.1)
    rails-assets-jquery-flot (0.8.3)
      rails-assets-jquery (>= 1.2.6)
    rails-assets-jquery-hoverIntent-jmalonzo (0.7.1)
      rails-assets-jquery (= 1.9.1)
    rails-assets-jquery-treetable (3.2.0)
      rails-assets-jquery (>= 1.6)
    rails-assets-smartmenus (0.9.7)
      rails-assets-jquery (>= 1.4.4)
    rails-assets-ui-autocomplete (0.5.7)
    rails-assets-underscore (1.7.0)
    rails-observers (0.1.2)
      activemodel (~> 4.0)
    rails_autolink (1.1.6)
      rails (> 3.1)
    railties (4.1.9)
      actionpack (= 4.1.9)
      activesupport (= 4.1.9)
      rake (>= 0.8.7)
      thor (>= 0.18.1, < 2.0)
    rainbow (2.0.0)
    rake (10.4.2)
    redis (3.2.0)
    redis-namespace (1.5.1)
      redis (~> 3.0, >= 3.0.4)
    ref (1.0.5)
    request_store (1.1.0)
    responders (1.1.2)
      railties (>= 3.2, < 4.2)
    rest-client (1.7.2)
      mime-types (>= 1.16, < 3.0)
      netrc (~> 0.7)
    roadie (3.0.2)
      css_parser (~> 1.3.4)
      nokogiri (~> 1.6.0)
    roadie-rails (1.0.4)
      rails (>= 3.0, < 4.3)
      roadie (~> 3.0)
    rspec-activemodel-mocks (1.0.1)
      activemodel (>= 3.0)
      activesupport (>= 3.0)
      rspec-mocks (>= 2.99, < 4.0)
    rspec-core (3.1.7)
      rspec-support (~> 3.1.0)
    rspec-expectations (3.1.2)
      diff-lcs (>= 1.2.0, < 2.0)
      rspec-support (~> 3.1.0)
    rspec-its (1.1.0)
      rspec-core (>= 3.0.0)
      rspec-expectations (>= 3.0.0)
    rspec-mocks (3.1.3)
      rspec-support (~> 3.1.0)
    rspec-rails (3.1.0)
      actionpack (>= 3.0)
      activesupport (>= 3.0)
      railties (>= 3.0)
      rspec-core (~> 3.1.0)
      rspec-expectations (~> 3.1.0)
      rspec-mocks (~> 3.1.0)
      rspec-support (~> 3.1.0)
    rspec-support (3.1.2)
    rubocop (0.28.0)
      astrolabe (~> 1.3)
      parser (>= 2.2.0.pre.7, < 3.0)
      powerpack (~> 0.0.6)
      rainbow (>= 1.99.1, < 3.0)
      ruby-progressbar (~> 1.4)
    ruby-oci8 (2.1.7)
    ruby-progressbar (1.7.1)
    ruby-rc4 (0.1.5)
    ruby2ruby (2.1.3)
      ruby_parser (~> 3.1)
      sexp_processor (~> 4.0)
    ruby_parser (3.5.0)
      sexp_processor (~> 4.1)
    rubyzip (1.1.6)
    rufus-scheduler (3.0.9)
      tzinfo
    safe_yaml (1.0.4)
    sanitize (3.1.0)
      crass (~> 1.0.1)
      nokogiri (>= 1.4.4)
      nokogumbo (= 1.2.0)
    sass (3.2.19)
    sass-rails (4.0.5)
      railties (>= 4.0.0, < 5.0)
      sass (~> 3.2.2)
      sprockets (~> 2.8, < 3.0)
      sprockets-rails (~> 2.0)
    selenium-webdriver (2.44.0)
      childprocess (~> 0.5)
      multi_json (~> 1.0)
      rubyzip (~> 1.0)
      websocket (~> 1.0)
    sexp_processor (4.4.5)
    sidekiq (3.3.0)
      celluloid (>= 0.16.0)
      connection_pool (>= 2.0.0)
      json
      redis (>= 3.0.6)
      redis-namespace (>= 1.3.1)
    sidekiq-cron (0.2.0)
      rufus-scheduler (>= 2.0.24)
      sidekiq (>= 2.17.3)
      tilt (< 2.0.0)
    simplecov (0.9.1)
      docile (~> 1.1.0)
      multi_json (~> 1.0)
      simplecov-html (~> 0.8.0)
    simplecov-html (0.8.0)
    sinatra (1.4.5)
      rack (~> 1.4)
      rack-protection (~> 1.4)
      tilt (~> 1.3, >= 1.3.4)
    slim (2.1.0)
      temple (~> 0.6.9)
      tilt (>= 1.3.3, < 2.1)
    slim-rails (2.1.5)
      actionpack (>= 3.0, < 4.2)
      activesupport (>= 3.0, < 4.2)
      railties (>= 3.0, < 4.2)
      slim (~> 2.0)
    slop (3.6.0)
    spring (1.2.0)
    sprockets (2.12.3)
      hike (~> 1.2)
      multi_json (~> 1.0)
      rack (~> 1.0)
      tilt (~> 1.1, != 1.3.0)
    sprockets-rails (2.2.4)
      actionpack (>= 3.0)
      activesupport (>= 3.0)
      sprockets (>= 2.8, < 4.0)
    sys-uname (0.9.2)
      ffi (>= 1.0.0)
    temple (0.6.10)
    term-ansicolor (1.3.0)
      tins (~> 1.0)
    terminal-table (1.4.5)
    therubyracer (0.12.1)
      libv8 (~> 3.16.14.0)
      ref
    thor (0.19.1)
    thread_safe (0.3.4)
    tilt (1.4.1)
    timecop (0.7.1)
    timers (4.0.1)
      hitimes
    tins (1.3.3)
    ttfunk (1.4.0)
    tzinfo (1.2.2)
      thread_safe (~> 0.1)
    uglifier (2.7.0)
      execjs (>= 0.3.0)
      json (>= 1.8.0)
    unf (0.1.4)
      unf_ext
    unf_ext (0.0.6)
    warden (1.2.3)
      rack (>= 1.0)
    webmock (1.20.4)
      addressable (>= 2.3.6)
      crack (>= 0.3.2)
    webrobots (0.1.1)
    websocket (1.2.1)
    websocket-driver (0.5.1)
      websocket-extensions (>= 0.1.0)
    websocket-extensions (0.1.1)
    wicked (1.1.0)
      rails (>= 3.0.7)
    will_paginate (3.0.7)
    wisper (1.6.0)
    wisper-activerecord (0.2.0)
      activerecord (>= 3.0.0)
      wisper (~> 1.3)
    wisper-sidekiq (0.0.1)
      sidekiq
      wisper
    xpath (2.0.0)
      nokogiri (~> 1.3)
    yui-compressor (0.12.0)

PLATFORMS
  ruby

DEPENDENCIES
  active_model_serializers (= 0.9.3)
  activerecord-oracle_enhanced-adapter (= 1.5.5)
  acts_as_list (= 0.6.0)
  ancestry (= 2.1.0)
  audited-activerecord (= 4.0.0)
  awesome_print (= 1.6.1)
  bcrypt (= 3.1.9)
  better_errors (= 2.1.1)
  bootstrap-sass (= 3.3.3)
  bootstrap-will_paginate (= 0.0.10)
  brakeman (= 3.0.0)
  cancancan (= 1.10.1)
  capistrano (= 2.15.5)
  capistrano-rails (= 1.0.0)
  capybara (= 2.4.4)
  capybara-webkit (= 1.4.0)
  ckeditor (= 4.1.1)
  coffee-rails (= 4.1.0)
  compass-rails (= 2.0.0)
  cucumber-rails (= 1.4.2)
  dalli (= 2.7.2)
  database_cleaner (= 1.4.0)
  dawnscanner (= 1.2.99)
  devise (= 3.4.1)
  devise-encryptable (= 0.2.0)
  devise_ldap_authenticatable (= 0.8.1)
  email_spec (= 1.6.0)
  exception_notification (= 4.0.1)
  factory_girl_rails (= 4.5.0)
  faraday (= 0.9.1)
  fastercsv (= 1.5.5)
  flip (= 1.0.1)
  font-awesome-sass (= 3.2.1.4)
  gon (= 5.2.3)
  headless (= 1.0.2)
  httpclient (= 2.6.0.1)
  jasmine (= 2.1.0)
  jquery-rails (= 3.1.2)
  jquery-ui-rails (= 4.2.1)
  js-routes (= 1.0.0)
  ladle (= 1.0.0)
  letter_opener (= 1.3.0)
  namae (= 0.9.3)
  net-scp (= 1.1.2)
  net-ssh (= 2.7.0)
  newrelic-redis (= 1.5.0)
  newrelic_rpm (= 3.9.9.275)
  ngannotate-rails (= 0.14.1)
  nokogiri (= 1.6.6.1)
  parser (= 2.2.0.2)
  pdf-inspector (= 1.2.0)
  phantomjs (= 1.9.8.0)
  poltergeist (= 1.5.1)
  princely (= 1.4.1)
  progressbar (= 0.21.0)
  pry (= 0.10.1)
  pry-byebug (= 2.0.0)
  rails (= 4.1.9)
  rails-assets-angular (= 1.2.26)
  rails-assets-angular-bootstrap (= 0.12.0)
  rails-assets-angular-mocks (= 1.2.26)
  rails-assets-angular-resource (= 1.2.26)
  rails-assets-angular-sanitize (= 1.2.26)
  rails-assets-asquera--prism (= 0.0.8)
  rails-assets-jquery-flot (= 0.8.3)
  rails-assets-jquery-hoverIntent-jmalonzo (= 0.7.1)
  rails-assets-jquery-treetable (= 3.2.0)
  rails-assets-smartmenus (= 0.9.7)
  rails-assets-ui-autocomplete (= 0.5.7)
  rails-assets-underscore (= 1.7.0)
  rails_autolink (= 1.1.6)
  rake (= 10.4.2)
  roadie-rails (= 1.0.4)
  rspec-activemodel-mocks (= 1.0.1)
  rspec-its (= 1.1.0)
  rspec-rails (= 3.1.0)
  rubocop (= 0.28.0)
  ruby-oci8 (= 2.1.7)
  rufus-scheduler (= 3.0.9)
  sanitize (= 3.1.0)
  sass (= 3.2.19)
  sass-rails (= 4.0.5)
  selenium-webdriver (= 2.44.0)
  sidekiq (= 3.3.0)
  sidekiq-cron (= 0.2.0)
  simplecov (= 0.9.1)
  sinatra (= 1.4.5)
  slim-rails (= 2.1.5)
  spring (= 1.2.0)
  therubyracer (= 0.12.1)
  timecop (= 0.7.1)
  uglifier (= 2.7.0)
  webmock (= 1.20.4)
  wicked (= 1.1.0)
  will_paginate (= 3.0.7)
  wisper-activerecord (= 0.2.0)
  wisper-sidekiq (= 0.0.1)
  yui-compressor (= 0.12.0)
```

**Comment by segiddins** [comment]
Well, did you try running with the `--full-index` flag as the error suggests doing?

**Comment by alexfalkowski** [comment]
Yes I did try running:

```ruby
bundle outdated --full-index
```

I get:

```ruby
Unknown switches '--full-index'
```

**Comment by alexfalkowski** [comment]
Eventually the command returns with the following error:

```ruby
Fetching gem metadata from https://rails-assets.org/...
Fetching gem metadata from https://rubygems.org/...Retrying dependency api due to error (2/3): Bundler::HTTPError Net::HTTPUnprocessableEntity: Too many gems (use --full-index instead)
Retrying dependency api due to error (3/3): Bundler::HTTPError Net::HTTPUnprocessableEntity: Too many gems (use --full-index instead)


Net::HTTPNotFound: <?xml version="1.0" encoding="UTF-8"?>
<Error><Code>NoSuchKey</Code><Message>The specified key does not
exist.</Message><Key>quick/Marshal.4.8/acts_as_user-1.1.1.gemspec.rz</Key><RequestId>9FC422DAC93EA9A0</RequestId><HostId>Jtiq0QIyekC2RiICrqJECSnA10UlSE/b08fBFT3rrM7oFZW0I96Dpfg36OVaKiOVLUrxcbjUZ+s=</HostId></Error>
```

**Comment by Nerian** [comment]
Since a Gemfile with a large number of gems is going to have those gems for the long term, can we make the `--full-index` option persist for that Gemfile so that we don't need to write it every-time?

Maybe we can use an option in the Gemfile, or a dot file.

**Comment by TimMoore** [comment]
Bundler should be using `--full-index` by default when there are 100 or
more gems. If that's not happening, there must be an edge case throwing it
off. More details would be helpful. If someone having this problem could
gist the output from `bundle install --verbose`, that would help.

**Comment by Nerian** [comment]
Sure, here it is: https://gist.github.com/Nerian/6ea104736101ad7bb103

**Comment by joecode** [comment]
same issue here, but --full-index fixed it.

**Comment by TrevorHinesley** [comment]
--full-index indeed fixes it, but there are PaaS that only allow `bundle install` (Cloud 66) as part of the build commands. Seems to make sense that Bundler should be able to determine how many gems need to be downloaded, and use --full-index on its own, without us explicitly using the flag, if need be.

**Comment by TimMoore** [comment]
This should be fixed in 1.7.13. Please try upgrading and let us know if you still have any problems.
